autolens/analysis/aggregator/aggregator.py

```diff
@@ -229,6 +174,261 @@ def fit_interferometer_via_database_from(
     )
 
 
+class AbstractAgg:

```

Comment:
If it's actually Abstract then inherit from ABC. Abstract classes should never be instantiated